kuzzle-esp32/01-rgb-led/rgb-led.md

```diff
@@ -0,0 +1,366 @@
+# USE KUZZLE AS AN IOT BACKEND WITH ESP32 MCU
+
+![kuzzle](img/35656a24-30e3-4241-a4c0-29e40c4e5b1c.jpg)
+
+## Intro: Use Kuzzle as an IoT backend With ESP32
+
+Kuzzle is an open-source backend suite that can be installed on-premises and includes a multi-protocol API that can be used to integrate IoT devices.
```

Comment:
Proposal to avoid the "that can" repetition: `Kuzzle is an open-source backend. It can be installed on-premises, and it features a multi-protocol API allowing to integrate IoT devices.`